### PR TITLE
Avoid page image jumps in fotorama gallery

### PR DESCRIPTION
**Pull request by daniloargentiero**
On mobile devises: when you scroll down on the product page and you are about half way of the product image the page jumps up to the top. Movement should be fluid always.  (copied from issue #10518)

### Description
This pull request makes movement fluid on mobile devices.

### Fixed Issues
1. magento/magento2#10518: Mobile product page image jumps

### Manual testing scenarios (copied from issue #10518)
1. Open a product page
2. Scroll down to approximately 50% of the image height and the page jumps back to the top. Issue also occurs when scrolling up
3. Bug is intermittent and glitchy and can take a few atrempts to reproduce. Mainly occurs when you use a flick style scroll past the 50% mark.

### Contribution checklist
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [X] All new or changed code is covered with unit/integration tests (if applicable)
 - [X] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by ihor-sviziev** [comment]
Hi @DaniloEmpire I tested your changes, unfortunately I still have issue that you've described on my Android device. Probably it caused by hiding top bar while you scrolling page down, page height is changing. Could you re-check it?

**Comment by daniloargentiero** [comment]
Hi @ihor-sviziev ,
sure, I'll check it asap.

**Comment by FreekVandeursen** [comment]
Hey @DaniloEmpire , we have just tried your changes, and it seems to have worked for us. We haven't seen the problem occur anymore since applying this change.
Thanks a lot!

**Comment by daniloargentiero** [comment]
Hi @ihor-sviziev ,
Please check again. It seems to work.
Thank you.

**Comment by ihor-sviziev** [comment]
@DaniloEmpire I recorded video where I'm reproducing this issue. It is reproducing much harder, but issue is still there

https://photos.app.goo.gl/HLsPKgnmDR1UHUVK2

**Comment by daniloargentiero** [comment]
Hi @ihor-sviziev ,
can you re-try?

**Comment by ihor-sviziev** [comment]
@magento-engcom-team give me new test instance

**Comment by magento-engcom-team** [comment]
Hi @ihor-sviziev, here is your [new test Magento instance](http://pr-14282.engcom.dev.magento.com)

File: app/code/Magento/Catalog/view/frontend/templates/product/view/gallery.phtml [hunk 1]
```diff
@@ -12,7 +12,7 @@
  * @var $block \Magento\Catalog\Block\Product\View\Gallery
  */
 ?>
-<div class="gallery-placeholder _block-content-loading" data-gallery-role="gallery-placeholder">
+<div class="gallery-placeholder _block-content-loading" data-gallery-role="gallery-placeholder" style="min-height: <?= /* @escapeNotVerified */ $block->getImageAttribute('product_page_image_medium', 'height') ?>">
```
Comment (ihor-sviziev) [inline app/code/Magento/Catalog/view/frontend/templates/product/view/gallery.phtml:15]: This value should be escaped

**Review by ihor-sviziev — CHANGES_REQUESTED** [review]
Hi,
One think looks not good enough. Will test this solution and will give you feedback

**Comment by ihor-sviziev** [comment]
Hi @DaniloEmpire,
I tested it again, still have this issue on my Android device, I have the same steps to reproduce and the same reproducing rate as previous time.

I think main issue - chrome's top navigation that hides once you're scrolling down the page. What if we will check only device width?

You can try to reproduce it here: http://pr-14282.engcom.dev.magento.com/test.html

**Comment by ihor-sviziev** [comment]
Hi @DaniloEmpire , are you interested in finalizing your PR?

**Comment by sidolov** [comment]
@DaniloEmpire , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration!

**Comment by amjadm61** [comment]
@magento give me test instance

**Comment by magento-engcom-team** [comment]
Hi @amjadm61. Thank you for your request. I'm working on Magento instance for you

**Pull request closed without merging**